isis/src/control/objs/BundleUtilities/AbstractBundleObservation.cpp

```diff
@@ -175,9 +175,10 @@ namespace Isis {
    *   @todo initParameterWeights() doesn't return false, so this methods always
    *         returns true.
    */
-//  bool AbstractBundleObservation::setSolveSettings(BundleObservationSolveSettings solveSettings) {
+  bool AbstractBundleObservation::setSolveSettings(BundleObservationSolveSettings solveSettings) {
     // different for both
-//  }
+    return false;
+  }
```

Comment:
We don't want this function as it will be different for both types. This should only exist in BundleObservation.

Reply:
Good point. I'll take it out.